agents/unix/conf/base/conf_netconsole.c

```diff
@@ -235,12 +274,12 @@ configure_netconsole(in_port_t local_port, const char *remote_host_name,
              "failed to obtain string representation of remote "
              "hardware address",
              "%02hhx:%02hhx:%02hhx:%02hhx:%02hhx:%02hhx",
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[0]),
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[1]),
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[2]),
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[3]),
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[4]),
-             0xff & (int)(remote_hwaddr_req.arp_ha.sa_data[5]));
+             0xff & (int)(remote_hwaddr.sa_data[0]),
+             0xff & (int)(remote_hwaddr.sa_data[1]),
+             0xff & (int)(remote_hwaddr.sa_data[2]),
+             0xff & (int)(remote_hwaddr.sa_data[3]),
+             0xff & (int)(remote_hwaddr.sa_data[4]),
+             0xff & (int)(remote_hwaddr.sa_data[5]));
```

Comment:
As you're touching it anyway: we have `TE_PRINTF_MAC_FMT` and `TE_PRINTF_MAC_VAL`